recipes/pysolr/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pysolr" %}
+{% set version = "3.6.0" %}
```

Comment:
Strangely, the `__version__` in the package from pypi.io says 3.5.0 (although the tarball is labeled as 3.6.0)